### DISCUSSION TITLE
Add option for specific task to be ignored from rerun?

### DISCUSSION THREAD

**Discussion by malkomalko** [zed-industries/zed#22522] [Feature Requests]
I have some global tasks that I use for opening specific files, that I want to fire as a one off and don't want it included when I re-run. It'd be nice to add an option in each task to `"ignore_from_rerun": true` or something similar.

Thoughts?

**Comment by B4Dmonkey** [comment]
this makes sense, i recently came across a small annoyance where i want to run a script based on the file and line number. it seems that when you rerun a task it doesn't update to use the current information

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
